process/utilities/errorlist.json

```diff
@@ -878,12 +878,12 @@
     {
       "no": 174,
       "level": 3,
-      "message": "CONVERT_PLASMOD2PROCESS: PLASMOD Transport model (ipedestal=3) has not converged"
+      "message": "OBSOLETE"
```

Comment:
I suspect we can actually removed these entries from the json file entirely since they should never be called, and if they were it would raise an error (probably what we want).